gcloud/connection.py

```diff
@@ -22,11 +22,8 @@
 class Connection(object):
     """A generic connection to Google Cloud Platform.
 
-    Subclasses should understand
-    only the basic types
-    in method arguments,
-    however they should be capable
-    of returning advanced types.
+    Subclasses should understand only the basic types in method arguments,
```

Comment:
How is this "docstring poetry" still here?  I swear I have removed it at least twice.